Demo/HelloWorld/Client/src/Program.cs

```diff
@@ -0,0 +1,212 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2023. All rights reserved.
+//   W. Kirschenmann   <wkirschenmann@aneo.fr>
+//   J. Gurhem         <jgurhem@aneo.fr>
+//   D. Dubuc          <ddubuc@aneo.fr>
+//   L. Ziane Khodja   <lzianekhodja@aneo.fr>
+//   F. Lemaitre       <flemaitre@aneo.fr>
+//   S. Djebbar        <sdjebbar@aneo.fr>
+//   J. Fonseca        <jfonseca@aneo.fr>
+//   D. Brasseur       <dbrasseur@aneo.fr>
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY, without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using ArmoniK.Api.Client.Options;
+using ArmoniK.Api.Client.Submitter;
+using ArmoniK.Api.gRPC.V1;
+using ArmoniK.Api.gRPC.V1.Submitter;
+
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+using static System.Console;
+
+
+namespace ArmoniK.Samples.HelloWorld.Client;
+
+internal class Program
+{
+  public static async Task<int> Main(string[] args)
+  {
+    // Define the options for the application with their description and default value
+    var endpoint = new Option<string>(name: "--endpoint",
+                                      description: "Endpoint for the connection to ArmoniK control plane.",
+                                      getDefaultValue: () => "http://localhost:5001");
+    var partition = new Option<string>(name: "--partition",
+                                       description: "Name of the partition to which submit tasks.",
+                                       getDefaultValue: () => "default");
+    var input = new Option<string>(name: "--input",
+                                   description: "Input string for the tasks.",
+                                   getDefaultValue: () => "Hello");
+    var nTasks = new Option<int>(name: "--nTasks",
+                                 description: "Number of tasks to send to ArmoniK.",
+                                 getDefaultValue: () => 1);
+    var nResultsPerTask = new Option<int>(name: "--nResultsPerTask",
+                                          description: "Number of results per tasks.",
+                                          getDefaultValue: () => 1);
```

Comment:
Do we really need this in the Hello World? (especially the multiple results per task)